added a query filter called “title” to API

ElliotStaudt opened the pull request:
Changed:
/backend/app/api/dependencies/items.py
/backend/app/models/schemas/items.py

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
@vanessa-cooper

ElliotStaudt committed:
add a query filter called “title” to API

Changed:
/backend/app/api/dependencies/items.py
/backend/app/models/schemas/items.py

vanessa-cooper commented:
Lovely. Let me take a look.

vanessa-cooper reviewed (changes requested):
Umm… something is not quite right, please take another look and try again.

ElliotStaudt committed:
Added table for titles and query calls.

Updated:
backend/app/...
- api/routes/items/items_recource.py
- db/...
  - migrations/versions/fdf8821871d7_main_tables.py
  - queries/tables.py
  - repositories/items.py

ElliotStaudt committed:
Testing if changing title query makes a difference
